public/sketch.js

```diff
@@ -1,29 +1,27 @@
 const socket = io.connect('http://localhost:3000');
 
 function setup(){
-<<<<<<< HEAD
-	createCanvas(1000, 700);
-=======
+	cursor(CROSS);
 	const canvas = createCanvas(500, 500);
 	canvas.parent('canvas-container');
->>>>>>> e565f2c0cfe6af105a3481530a9397f75726ad4b
 	background(100);
-	socket.on('mouse', newDrawing);	
+	socket.on('mouse', newDrawing);
 }
 
 function draw(){
-	stroke(255);
-	fill(255);
-	line(mouseX, mouseY, pmouseX, pmouseY);
-	mouseDataSend();
 }
 
 function newDrawing(data) {
 	stroke(255, 0, 100);
-	console.log(data);
 	line(data.x, data.y, data.px, data.py);
 }
 
+function mouseDragged(){
```

Comment:
is this function called when the mouse is being pressed down and dragged? if so, a better name might be "onMouseDrag". if it's just a function that is called when the mouse is moved over the canvas like before, convention might call it "onMouseMove"

Reply:
'mouseDragged()' is a built-in function to P5.js that is called when the mouse is clicked and dragged. Seemed like the path of least resistance...?

Do you think it would be best to tackle a custom function?

Reply:
Ahhhh ok. Nope. Ima merge 👍